tests/pytorch/nightly/hf-glue.libsonnet

```diff
@@ -222,11 +137,7 @@ local utils = import 'templates/utils.libsonnet';
     accelerator: tpus.v4_8,
   },
   configs: [
-    hf_glue + v2_8 + bert_base_cased + timeouts.Hours(2) + xrt,
-    hf_glue + v3_8 + xlnet_large_cased + timeouts.Hours(5) + xrt,
-    hf_glue + v3_8 + roberta_large + timeouts.Hours(8) + xrt,
     hf_glue + v3_8 + distilbert_base_uncased + timeouts.Hours(2) + xrt,
```

Comment:
should we just change this to pjrt?

Reply:
Do we need to test 4 different models on the same task? IMO we should just pick one `BERT` and test that on both runtimes